vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -2,150 +2,166 @@
 
-## ent bundle Command List 
-| Commands | Subcommands|  Description
+## Docker-based Bundle Commands
+| Commands | Subcommands |  Description |
 |:-|:-|:----------------------------------
-|`ent bundle build`||Build components (MFE, MS) with a selector
-|`ent bundle generate-cr`||Generate the Entando Custom Resource for a bundle project|
-|`ent bundle help` ||Display help for ent bundle
-|`ent bundle info`||Show status information for the bundle project 
-|`ent bundle init`||Initialize project folder structure and descriptor
-|`ent bundle list`|| List the available components in the bundle
-|`ent bundle mfe` |	`add`	 | Add an MFE project component
-| |	`rm` |	Remove an MFE project component 	
-|`ent bundle ms`|`add`| Add an MS project component		 	
-| |	`rm` |	Remove an MS project component
-|`ent bundle run`|| Run bundle components 
-|`ent bundle pack`||Create distribution artifacts (Docker images)
-|`ent bundle publish`||Publish images to a Docker registry
-|`ent --debug bundle`||Enable debug mode|
-
-## Initialization 
+|`ent bundle build`| | Build components (MFE, MS) with a selector |
+|`ent bundle generate-cr`| | Generate the Entando Custom Resource for a bundle project |
+|`ent bundle help` | | Display help for ent bundle |
+|`ent bundle info`| | Show status information for the bundle project |
+|`ent bundle init`| | Initialize the project folder structure and descriptor |
+|`ent bundle list`| | List the available bundle components |
+|`ent bundle mfe` |	`add` | Add a micro frontend |
+| |	`rm` |	Remove a micro frontend |
+|`ent bundle ms`|`add`| Add a microservice |	 	
+| |	`rm` |	Remove a microservice |
+|`ent bundle run`| | Run bundle components |
+|`ent bundle pack`| | Create distribution artifacts (Docker images) |
+|`ent bundle publish`| | Publish images to a Docker registry |
+|`ent --debug bundle`| | Enable debug mode |
+
+### Initialization 
 
 | Command |  Description
 |:--|:--
-|`ent bundle init [name]` |Initialize a new empty project with the default structure and files
-|`ent bundle init [name] --from-hub`| Initialize a bundle from an Entando Hub
+|`ent bundle init [name]` | Initialize a new empty project with the default structure and files |
+|`ent bundle init [name] --from-hub`| Initialize a bundle from the Entando Cloud Hub |
+|`ent bundle init [name] --from-hub --hub-url=[url]` | Initialize a bundle from a custom Entando hub |
 
 #### Command Details
-* `ent bundle init [name] --from-hub`: Utilize an existing bundle from an Entando Hub to jumpstart your project. The ent bundle tool will pull the package and rebuild the structure as needed, ready for customization. 
+The option `--from-hub` leverages an existing bundle from an Entando hub to jumpstart your project. The ent bundle tool will pull the package and rebuild the structure, after which it can be customized locally. If the command does not include `--hub-url`, ent will default to use the Entando Cloud Hub. Include the option `--hub-url` to specify a custom hub.
```

Comment:
there is no Entando Hub entity. there is the Local Hub, a private hub (naming/capitalization tbd) and the Entando Cloud Hub. a hub that is neither the Local Hub nor an Entando Cloud Hub is an Entando hub

Reply:
walter's comment on the google doc kind of solidified for me that this is how we're treating all flavors of Entando hub, but i think we need to talk about it with nathan, too. idk if this will/should change.

Reply:
changed per standup